frontend/ui/src/app/tutorials/agents/agents-tutorial.component.html

```diff
@@ -0,0 +1,330 @@
+          <cdk-step label="Welcome" [stepControl]="welcomeFormGroup">
+            <div class="px-20 py-8 w-full">
+              <h3 class="mb-4 text-xl font-semibold text-gray-900 dark:text-white">
+                Agents, Applications and Deployments
+              </h3>
+
+              <div class="text-gray-900 dark:text-white mb-12">
+                <p>
+                  In this tutorial, you will learn how to set up the Distr agent deploy a Docker Compose app with it.
+                </p>
+                <p>
+                  We have already prepared a sample application called
+                  <a href="https://github.com/glasskube/hello-distr" class="text-primary-600 underline" target="_blank"
+                    >hello-distr</a
+                  >.
+                </p>
+              </div>
+
+              <div class="gap-4 sm:flex sm:items-center sm:justify-between">
+                <button
+                  (click)="navigateToOverviewPage()"
+                  class="flex-1 sm:flex-none items-center justify-center w-full rounded-lg border border-gray-200 bg-white px-5 py-2.5 text-sm font-medium text-gray-900 hover:bg-gray-100 hover:text-primary-700 focus:z-10 focus:outline-none focus:ring-4 focus:ring-gray-100 dark:border-gray-600 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white dark:focus:ring-gray-700 sm:w-auto">
+                  Exit Tutorial
+                </button>
+                <button
+                  (click)="continueFromWelcome()"
+                  type="submit"
+                  class="flex w-full items-center justify-center rounded-lg border border-primary-700 bg-primary-700 px-5 py-2.5 text-sm font-medium text-white hover:border-primary-800 hover:bg-primary-800 focus:outline-none focus:ring-4 focus:ring-primary-300 dark:border-primary-700 dark:bg-primary-600 dark:hover:border-primary-700 dark:hover:bg-primary-700 dark:focus:ring-primary-800 sm:mt-0 sm:w-auto">
+                  Let's Go
+                  <fa-icon class="ml-2" [icon]="faArrowRight"></fa-icon>
+                </button>
+              </div>
+            </div>
+          </cdk-step>
+          <cdk-step label="Deploy hello-distr" [stepControl]="deployFormGroup">
+            <div class="p-8 w-full">
+              <form [formGroup]="deployFormGroup" (ngSubmit)="continueFromDeploy()">
+                <ol
+                  class="ml-4 relative text-gray-900 dark:text-white mb-12 border-s border-gray-200 dark:border-gray-700">
+                  <li class="mb-10 ms-8">
+                    <span
+                      class="absolute flex items-center justify-center w-8 h-8 bg-gray-100 rounded-full -start-4 ring-4 ring-white dark:ring-gray-900 dark:bg-gray-700">
+                      <fa-icon [icon]="faLightbulb"></fa-icon>
+                    </span>
+                    <h3 class="text-lg font-medium leading-tight">Deploy hello-distr</h3>
+                    <div
```

Comment:
This looks off in dark mode. Maybe we can change it to [this one](https://flowbite.com/docs/components/alerts/#bordered-alerts)?

Also maybe only show it on MacOS (can use the [CDK platform](https://material.angular.io/cdk/platform/overview) for this)

Reply:
I see where you're coming from but IMO it's still better :sweat_smile: